dev-infra/release/stamping/env-stamp.ts

```diff
@@ -38,26 +42,30 @@ function hasLocalChanges() {
 }
 
 /**
- * Get the version for generated packages.
+ * Get the versions for generated packages.
  *
  * In snapshot mode, the version is based on the most recent semver tag.
  * In release mode, the version is based on the base package.json version.
  */
-function getSCMVersion(mode: EnvStampMode) {
+function getSCMVersions(mode: EnvStampMode) {
   const git = GitClient.get();
   if (mode === 'release') {
     const packageJsonPath = join(git.baseDir, 'package.json');
-    const {version} = require(packageJsonPath);
-    return version;
+    const {version} = new SemVer(require(packageJsonPath).version);
+    const {version: experimentalVersion} = createExperimentalSemver(new SemVer(version));
+    return [version, experimentalVersion];
```

Comment:
Nit: I think it would be better to use an object literal compared to returning an arbitrary-sized array:

```suggestion
    return {version, experimentalVersion};
```